appveyor.yml

```diff
@@ -2,7 +2,7 @@ version: '{build}'
 branches:
```

Comment:
Since we're adding GHA we don't have any benefits of additionally building on AppVeyor and so should remove this file.